.coveragerc

```diff
@@ -1,11 +1,30 @@
+# Generated from:
```

Comment:
I am confused. Weren't we killing off .coveragerc?

Reply:
Generally, yes. But running coverage using multiple processes requires to have a `.coveragerc` it does not read its configuration from `tox.ini` in this case. So we have to keep `.coveragerc` for buildout recipes and packages which run tests in separate processes.